apps/scan/frontend/src/screens/election_manager_screen.tsx

```diff
@@ -376,6 +379,10 @@ export function ElectionManagerScreen({
 /* istanbul ignore next */
 export function DefaultPreview(): JSX.Element {
   const { electionDefinition } = usePreviewContext();
+  // eslint-disable-next-line @typescript-eslint/require-await
+  const doExportLogs: () => Promise<LogsResultType> = async () => {
+    return ok();
+  };
```

Comment:
Another mock to turn into `jest.fn()`

Reply:
this one's not a test, it's an actual default but maybe the issue here is we shouldn't have this be a reasonable default. I think there was a reason I did this but... let me see if I can undo it.

Reply:
oh wait.. it's a defaultpreview.... maybe it is a test? what's the right way to think about this? Like a test?

Reply:
Oh whoops, I didn't read closely. I think it's fine how you have it

Reply:
This is for the "preview mode" where you can view screens of the app directly by going to `/preview`